be/src/olap/rowset/segment_v2/inverted_index_compound_directory.cpp

```diff
@@ -512,11 +511,7 @@ void DorisCompoundDirectory::FSIndexOutput::close() {
 
 int64_t DorisCompoundDirectory::FSIndexOutput::length() const {
     CND_PRECONDITION(_writer != nullptr, "file is not open");
-    int64_t ret;
-    if (!_writer->fs()->file_size(_writer->path(), &ret).ok()) {
-        return -1;
-    }
-    return ret;
+    return _writer->bytes_appended();
```

Comment:
The default value of `_bytes_appended` is 0,
but the origin logic may return -1, need to check if `-1` means something to the caller.

Reply:
Seems that there is no code within clucene that checks if the return value of `length` is -1.